packages/trpc/server/routers/viewer/organizations/create.handler.ts

```diff
@@ -58,132 +54,88 @@ export const createHandler = async ({ input, ctx }: CreateOptions) => {
 
   if (hasAnOrgWithSameSlug || RESERVED_SUBDOMAINS.includes(slug))
     throw new TRPCError({ code: "BAD_REQUEST", message: "organization_url_taken" });
-  if (userCollisions) throw new TRPCError({ code: "BAD_REQUEST", message: "admin_email_taken" });
 
-  const password = createHash("md5")
-    .update(`${adminEmail}${process.env.CALENDSO_ENCRYPTION_KEY}`)
-    .digest("hex");
-  const hashedPassword = await hashPassword(password);
+  if (orgOwner) {
+    // Invite existing user as the owner of the organization
```

Comment:
I think this is still an outstanding TODO right?

Reply:
Actually this is the only case we are supporting right now. I will turn it into a negation to make it clear.